.github/workflows/master.yml

```diff
@@ -0,0 +1,149 @@
+name: master
+
+on:
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches: 
+      - master
+
+jobs:
+  python35:
+    runs-on: ubuntu-latest
+    env:
+      SPARK_VERSION: 2.3.4
+      PANDAS_VERSION: 0.23.4
+      PYARROW_VERSION: 0.10.0
+      KOALAS_USAGE_LOGGER: databricks.koalas.usage_logging.usage_logger
+      QT_QPA_PLATFORM: offscreen
```

Comment:
Can you add a comment for his variable per 66f5119?

Reply:
@HyukjinKwon actually we don't need the anymore since that issue is resolved by `xvfb-run`.

FYI.

https://github.com/databricks/koalas/blob/aebfd56812dd16a41e388f7a97eb1ce7530238f7/dev/pytest#L41-L42